Zend/zend_closures.stub.php

```diff
@@ -6,13 +6,17 @@ final class Closure
 {
     private function __construct() {}
 
-    public static function bind(Closure $closure, ?object $newthis, $newscope = UNKNOWN): ?Closure {}
+    /** @param object|string|null $newScope */
+    public static function bind(Closure $closure, ?object $newThis, $newScope = UNKNOWN): ?Closure {}
```

Comment:
Sadly, I couldn't use the `object|string|null` ZPP here :( since unfortunately, there's distinction between when the `$newScope` is `null` and when the argument is not passed. I can't see any elegant solution to the problem.